### DISCUSSION TITLE
Recurring "There's nothing to scrape" error

### DISCUSSION THREAD

**Discussion by fightcancer2** [UltimaHoarder/UltimaScraper#1049] [Q&A]
ISSUE RESOLVED by installing the latest version.  Thank you!!

(This situation is different than the other thread with a similar name as evidence by the screenshot.  In my bug, I don't even get to the "Choose Profile" line.)

As soon as I activate the script with G:\GitHub\start_ofd.py, then I get this recurring error (see screenshot).  It's been happening for a couple weeks.

I just updated my ..\.profiles\OnlyFans\default\auth.json file with sess, auth_id, app-token, and user-agent.

What am I missing please?

![Image1](https://user-images.githubusercontent.com/70211007/115885751-db925180-a415-11eb-9917-ac422a77f57d.png)

**Comment by LunarPenguin** [comment]
if the problems is solved please close / mark the discussion as answered

**Reply by fightcancer2** [reply]
> if the problems is solved please close / mark the discussion as answered

Not sure how to close / mark it answered.
